schema.graphql

```diff
@@ -120,3 +302,286 @@ type GenesisFile @entity {
   id: ID!
   raw: String!
 }
+
+type MsgStakeSupplier @entity {
+  id: ID!
+  signer: Account!
+  supplier: Supplier!
+  owner: Account!
+  # we are not using Coin because we want to be able to query and sum by amount
+  stakeAmount: BigInt!
+  stakeDenom: String!
+  block: Block!
+  transaction: Transaction!
+  message: Message!
+  serviceConfigs: [MsgStakeSupplierService] @derivedFrom(field: "stakeMsg")
```

Comment:
1. I think this should be the service config itself.
2. I think it should be required.

```suggestion
  serviceConfigs: [SupplierServiceConfig]! @derivedFrom(field: "supplier")
```